README.md

```diff
@@ -61,4 +61,4 @@ To utilize this function, you will need to manually run the deployment script.
 We will eventually have a Jenkins job that will run the script whenever ingest is updated.
 Also if you would like to run a new script you will need to modify the script that is being called in deployment script and bump the version.
 
-The script will only run if the version is a newer version of the db, the script also create the collection for the version if it doesn't exist yet.
+The script will only run if the version is a newer version of the db, the script also create the collection for the version if it doesn't exist yet. Set your .env file to point to the instances of the TA1 servers you have running locally. 
```

Comment:
I was able to use `https://darpaitm.caci.com/soartech/` and `https://darpaitm.caci.com/adept/`, which is handy if you haven't or don't want to run local TA1 servers.  Of course it requires that those servers are running...

Reply:
I think we orignially had those in the .env but they didn't work on production. I can add them back in just commented out for use locally.

Reply:
Oh right-- yes it should definitely be commented out.  It should work on production by default.